tenets/codelingo/code-review-comments/go-error-fmt/codelingo.yaml

```diff
@@ -3,9 +3,26 @@ funcs:
     type: asserter
     body: |
       function (a) {
-        var dot = a.charAt(a.length - 1) === '.'
-        var cap = a.charAt(0) >= "A" && a.charAt(0) <= "Z"
-        return dot || cap
+        if (a.charAt(a.length - 1) === '.') {
+          return true
+        }
+
+        var first = a.split(" ")[0]
+        if (first.substring === first.toUpperCase()) {
+          return false
+        }
+
+        // We ignore errors with words containing capitals anywhere in first word
+        // unless the only capital is at the start. This prevents acronyms or
+        // (most) code-related words being caught
+        first = first.substring(1)
```

Comment:
I'm confused (probably because I don't know JS 😛 ) but is `first.substring` a function, as it appears to be here, or is it a character property, as it appears to be on line 11?

Reply:
Whoops. No idea how substring ended up on line 11. It is a string function. Fixed